### PR TITLE
simplify/unify API

### PR DESCRIPTION
**Pull request by turboMaCk**
Some additional simplification/unification of public API before release.

### CODE REVIEW

File: src/WeakCss.elm [hunk 1]
```diff
@@ -73,8 +72,8 @@ namespace name =
     --> class "menu__list"
 
 -}
-addElement : String -> ClassName -> ClassName
```
Comment (kraklin) [inline src/WeakCss.elm]: You should update the doc tests

Comment (turboMaCk) [reply src/WeakCss.elm]: yep already done

Comment (kraklin) [reply src/WeakCss.elm]: Great, now just increase the version and we are ready to go

**Comment by turboMaCk** [comment]
@kraklin yeah we will go to 2.0.0. In theory, we can even keep 1.0.0 with a new version of elm but I don't want to mess git tags. `addElement` vs `addMany` was too odd API.

**Comment by kraklin** [comment]
@turboMaCk totally agree. It was confusing. Should we increase the version within this PR or separately?

File: src/WeakCss.elm (hunk 1 above)
Comment (kraklin) [reply src/WeakCss.elm]: At least you now know that tests are really running on CI

**Pull request merged**